### ISSUE TITLE
PS5 freezing / shutting down with Libhijacker enabled

### ISSUE THREAD

**Issue by mmcgr97** [illusion0001/libhijacker#32]
Does anyone know how to fix Libhijacker randomly freezing and shutting down my PS5 when launching / playing games?

I've tried jailbreaking my launch model PS5 on both 4.51 and 4.50 firmware with both idlesauce and zecoxao and etahen and with libhijacker enabled (config.ini set to Libhijacker_cheats=1") my PS5 regularly either freezes or shuts down when games are starting up or they've been open for a few minutes. Seems to happen on both games that can be FPS boosted with libhijacker and games that don't work with it.

Any solution would be greatly appreciated

**Comment by jackfavvv** [comment]
Don't enable cheats or the Libhijacker though etaHEN or by the config.ini from Itemzflow.

**Comment by mmcgr97** [comment]
> Don't enable cheats or the Libhijacker though etaHEN or by the config.ini from Itemzflow.

So then how do I enable it?

**Comment by mmcgr97** [comment]
> Don't enable cheats or the Libhijacker though etaHEN or by the config.ini from Itemzflow.

So then how am I supposed to enable 60fps patches?

**Comment by jackfavvv** [comment]
> > Don't enable cheats or the Libhijacker though etaHEN or by the config.ini from Itemzflow.
> 
> So then how am I supposed to enable 60fps patches?

There's an updated payload on Idlesauce's webhost. First pick k-stuff, then Libhijacker and it will automatically apply the patches when you launch the game.

**Comment by mmcgr97** [comment]
Tried it and RDR2 is still crashing (although not as much) as with ETAhen

**Comment by jackfavvv** [comment]
> Tried it and RDR2 is still crashing (although not as much) as with ETAhen

I'm currently playing Batman: Arkham Knight with just k-stuff and Libhijacker payload and everything runs fine. Have you tried other games, or is this only happening with RDR2?

**Comment by mmcgr97** [comment]
Just RDR2 seems to have the problem now, where as I couldn't play more than 5 minutes of Far Cry 3 or Slender Man with ETAHEN 1.8b. Still crashes with libhijacker and kstuff although far less often.

Do you know why ETAHEN is causing games to crash?

**Comment by mmcgr97** [comment]
> > Tried it and RDR2 is still crashing (although not as much) as with ETAhen
> 
> I'm currently playing Batman: Arkham Knight with just k-stuff and Libhijacker payload and everything runs fine. Have you tried other games, or is this only happening with RDR2?

Just RDR2 seems to have the problem now, where as I couldn't play more than 5 minutes of Far Cry 3 or Slender Man with ETAHEN 1.8b. Still crashes with libhijacker and kstuff although far less often.

Do you know why ETAHEN is causing games to crash?

**Comment by mmcgr97** [comment]
Really annoying bug I've found with Kstuff + Libhijacker. Libhijacker randomly stops working after switching to a new game. Sometimes I can switch between several games before it stops working and other times it stops working after just 1 game. Button combinations (Square + Triangle, Square + Circle) stop working too.

Any idea why this is happening? Doesn't happen with ETAHEN.

**Comment by mmcgr97** [comment]
Tried Arkham Knight with Kstuff and Libhijacker and it freezes at the opening level. Even without libhijacker and the game running in 30fps it freezes. PS5 doesn't turn off though

**Comment by jackfavvv** [comment]
> Do you know why ETAHEN is causing games to crash?

Yeah, but it's hard to explain. In short though it has come conflicting issues with the Debug. Which was why before, if you mess with the values in the config.ini from Itemzflow, your system will crash the next time you click on etaHEN and it does it debug. 



> Tried Arkham Knight with Kstuff and Libhijacker and it freezes at the opening level. Even without libhijacker and the game running in 30fps it freezes. PS5 doesn't turn off though

Are you talking about the diner scene where Batman makes his appearance on top of that skyscraper? 
Yeah, I had the exact same issue at that exact spot. It pretty means you have a bad dump of the game or update. Here try the one I'm using, CUSA00133 (v1.15) by FantasmaXDS. It works flawlessly.

**Comment by mmcgr97** [comment]
That's the one I used and it still froze. I'll download it again just to make sure

Some of the lit up signs were also flickering during that upwards panning shot where Batman first appears, which doesn't happen on my retail PS5.

Very weird.

**Comment by mmcgr97** [comment]
Do you know if ETAHEN 1.8b is using the latest version of libhijacker?

**Comment by jackfavvv** [comment]
> That's the one I used and it still froze. I'll download it again just to make sure
> 
> Some of the lit up signs were also flickering during that upwards panning shot where Batman first appears, which doesn't happen on my retail PS5.
> 
> Very weird.

That is weird, because I've even tested that exact spot five time in a row when Batman showed up and it no longer froze. There were some light flickering, though. Anyways, I recommend you trying again. With just k-stuff + Libhijacker. Don't use the feature from Itemzflow, because that will freeze you PS5 for sure. 




> Do you know if ETAHEN 1.8b is using the latest version of libhijacker?

Nope, that you'll have to ask LightningMods.

**Comment by mmcgr97** [comment]
Kstuff + Libhijacker is what I tried it with. I even tried it with only kstuff at 30fps and it still crashed

**Comment by jackfavvv** [comment]
> Kstuff + Libhijacker is what I tried it with. I even tried it with only kstuff at 30fps and it still crashed

That means that you tried the game a t neutral, and it still crashed. Which means that you have a bad, corrupted copy.
Anyways, I've tested it 6 times in a row now and it works just fine(check video). 
https://youtu.be/PMbdT72bKDc

**Comment by mmcgr97** [comment]
I've tried several different version of Arkham Knight from DLPS and all of them freeze on PS5 with or without Libhijacker but all work perfectly on my jailbroken PS4 Pro.

**Comment by jackfavvv** [comment]
I'll lend you my copy, if you want. It has all the DLC merged together, so I can be played on the PS5 as well.

**Comment by jackfavvv** [comment]
....no. That one isn't mine. My version is CUSA00133 and it's just 1 pkg file with everything in it. The one from your link has 2 pkg files.

**Comment by jackfavvv** [comment]
LOL, it's like a 50gib pkg file. And I haven't even uploaded it yet.

**Comment by jackfavvv** [comment]
Forgot to ask, but what firmware is your PS5 on? And are you playing the game internally or externally with a portable drive?

**Comment by mmcgr97** [comment]
4.51 and tested on both internal and m2 drive

**Comment by jackfavvv** [comment]
I'm on 4.50, so I'm not sure if our experience are going to be the exact same. Anyways, for now try this, in idlesauce's webhost, scroll all way down and use, PS5_Browser_appCache_remove to clear your cache. Then reopen the page and to auto cache everything again. This should free up some memory.

**Comment by mmcgr97** [comment]
Tried that. No difference

**Comment by jackfavvv** [comment]
Do you still have the PS5 on FW 4.50?

**Comment by mmcgr97** [comment]
No it's on 4.51. Don't think there's a difference

**Comment by jackfavvv** [comment]
Well, if my PKG doesn't work for you, then THAT will be the difference.

**Comment by pixeltester** [comment]
Could you geniuses perhaps NOT arrange piracy on Illusion's github repositories by any chance? Thanks

**Comment by mmcgr97** [comment]
> The file is ready. What is your email?

mmcgr97@gmail.com

**Comment by mmcgr97** [comment]
> Could you geniuses perhaps NOT arrange piracy on Illusion's github repositories by any chance? Thanks

Does owning a legit copy on my PSN account make it ok?

**Comment by mmcgr97** [comment]
> Could you geniuses perhaps NOT arrange piracy on Illusion's github repositories by any chance? Thanks

Sorry 😂😂😂
